prov/rxd/src/rxd_ep.c

```diff
@@ -580,6 +584,14 @@ static int rxd_ep_send_op(struct rxd_ep *rxd_ep, struct rxd_x_entry *tx_entry)
 
 	pkt_entry->pkt_size = tx_entry->bytes_done + sizeof(*op) + rxd_ep->prefix_size;
 
+	if (tx_entry->cq_entry.flags & FI_RMA) {
+		for (op->iov_count = 0; op->iov_count < rma_count; op->iov_count++) {
+			op->rma[op->iov_count].addr = rma_iov[op->iov_count].addr;
+			op->rma[op->iov_count].len = rma_iov[op->iov_count].len;
+			op->rma[op->iov_count].key = rma_iov[op->iov_count].key;
+		}
```

Comment:
Can this use memcpy?

Reply:
Yup, added this in a follow up patch but I'll add it here instead to soothe your soul.